spring-cloud-task-integration-tests/src/test/java/org/springframework/cloud/task/launcher/TaskLauncherSinkTests.java

```diff
@@ -101,6 +103,22 @@ public void setup() {
 		properties.put("spring.datasource.username", DATASOURCE_USER_NAME);
 		properties.put("spring.datasource.password", DATASOURCE_USER_PASSWORD);
 		properties.put("spring.datasource.driverClassName", DATASOURCE_DRIVER_CLASS_NAME);
+		properties.put("spring.application.name",TASK_NAME);
+
+		JdbcTemplate template = new JdbcTemplate(this.dataSource);
+		template.execute("DROP TABLE IF EXISTS TASK_TASK_BATCH");
+		template.execute("DROP TABLE IF EXISTS TASK_SEQ");
+		template.execute("DROP TABLE IF EXISTS TASK_EXECUTION_PARAMS");
+		template.execute("DROP TABLE IF EXISTS TASK_EXECUTION");
+		template.execute("DROP TABLE IF EXISTS BATCH_STEP_EXECUTION_SEQ");
+		template.execute("DROP TABLE IF EXISTS BATCH_STEP_EXECUTION_CONTEXT");
+		template.execute("DROP TABLE IF EXISTS BATCH_STEP_EXECUTION");
+		template.execute("DROP TABLE IF EXISTS BATCH_JOB_SEQ");
+		template.execute("DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_SEQ");
+		template.execute("DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_PARAMS");
+		template.execute("DROP TABLE IF EXISTS BATCH_JOB_EXECUTION_CONTEXT");
+		template.execute("DROP TABLE IF EXISTS BATCH_JOB_EXECUTION");
+		template.execute("DROP TABLE IF EXISTS BATCH_JOB_INSTANCE");
```

Comment:
Because of the timing involved, I won't object to this, but we should clean this up in the future.